BendAssist.App/BendAssists/BendRelief.cs

```diff
@@ -5,124 +5,106 @@ namespace BendAssist.App.BendAssists;
 
 #region class BendRelief --------------------------------------------------------------------------
 public sealed class BendRelief : BendAssist {
-    #region Constructors --------------------------------------------
-    public BendRelief (Part part) => (mPart, mPLines) = (part, part.PLines);
-    #endregion
+   #region Constructors --------------------------------------------
+   public BendRelief (Part part) => (mPart, mPLines) = (part, part.PLines);
+   #endregion
 
-    #region Methods -------------------------------------------------
-    /// <summary>Method to asses whether bend relief necessary for slected base and bend line</summary>
-    public override bool Assisted () {
-        return mCanAssist;
-    }
+   #region Methods -------------------------------------------------
+   /// <summary>Method to assess whether bend relief necessary for slected base and bend line</summary>
+   public override bool Assisted () {
+      return true;
+   }
 
-    public override void Execute () {
-        if (mPart != null) {
-            if (mPart.BendLines.Count != 0) {
-                var temp = mPart.Vertices.Where (x => IsRepeated (mPart.Vertices, x)).Where (x => IsWithinBound (x, mPart.Bound)).ToList ();
-                if (temp.Count > 0) {
-                    mCommonVertices = temp[..(temp.Count / 2)];
-                    mHLines = mPLines.Where (x => x.Orientation == EOrientation.Horizontal).ToList ();
-                    mVLines = mPLines.Where (x => x.Orientation == EOrientation.Vertical).ToList ();
-                    ApplyBendRelief (mPart);
-                }
-                return;
-            }
-        }
-    }
-    #endregion
+   /// <summary>Applies bend relief for the given part</summary>
+   public override void Execute () {
+      if (mPart != null && mPart.BendLines.Count != 0) {

```

Comment:
if (mPart is null || mPart.BendLines is null || mPart.BendLines.Count is 0) return;